notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Nit: Write in 2nd person, not first person.</p><p> </p><p>"Let's" is not recommended according to Google's writing style guide. <a href="https://developers.google.com/style/person" target="_blank">https://developers.google.com/style/person</a></p><p> </p><p>I suggest:</p><p> </p><p>Create a DataFrame in BigQuery DataFrames from local sample data. Use a list of lists to create a column with the dtype <code>list&lt;int64&gt;[pyarrow]</code>,  which corresponds to the <code>ARRAY&lt;INT64&gt;</code> type in BigQuery.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks!</p><p> </p><p> </p><p> </p>